<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_viewpoint</class>
 <widget class="QWidget" name="tab_viewpoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>68</width>
        <height>453</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="ImageAreaPreview" name="image_preview_widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="1">
      <widget class="QSpinBox" name="img_area_w">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>320</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Image Area W:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Loop Mode:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="init_y_spinBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>-999</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="add_Button">
       <property name="text">
        <string>Add New Entry</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QSpinBox" name="img_area_h">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Image Area H:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="delay_spinBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="name_textEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Image File:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>No Loop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Continuous Loop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auto-Reverse Loop</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Entry:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="init_x_spinBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>-999</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Dest X:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>View Final X:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="destx_spinBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>-999</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>View Initial Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="select_comboBox"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Blocking:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="blocking_checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="filename_comboBox"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Delay each frame: (ms)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QSpinBox" name="desty_spinBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>-999</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>View Final Y:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>View Initial X:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Dest Y:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="posx_spinBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>320</number>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QSpinBox" name="posy_spinBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Reset Size</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/toolbar_icons/task-recurring.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageAreaPreview</class>
   <extends>QWidget</extends>
   <header>scenes/imageareapreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
